esx_service/vmdk_ops_test.py

```diff
@@ -1086,23 +1096,32 @@ def test_vmdkops_on_different_tenants(self):
         # list volumes for tenant1
         opts = {}
         result = vmdk_ops.executeRequest(vm1_uuid, self.vm1_name, self.vm1_config_path, 'list', None, opts)
-       
         # listVMDK return vol name with the format like vol@datastore
         self.assertEqual(3, len(result))
-        self.assertEqual("tenant1_vol1@"+self.datastore_name, result[0]['Name'])
-        self.assertEqual("tenant1_vol2@"+self.datastore_name, result[1]['Name'])
-        self.assertEqual("tenant1_vol3@"+self.datastore_name, result[2]['Name'])
 
+        volume_names = ["tenant1_vol1@"+self.datastore_name,"tenant1_vol2@"+self.datastore_name,"tenant1_vol3@"+self.datastore_name]
+        for name in volume_names:
+            for j in range(len(result)):
+                if result[j]['Name'] == name:
+                   break
+                if j+1 == len(result):
+                   logging.error("Cannot find volume %s", result[j]['Name'])
+                   self.assertFalse(True)
+       
         # list volumes for tenant2
         opts = {}
-        result = vmdk_ops.executeRequest(vm2_uuid, self.vm2_name, self.vm2_config_path, 'list', None, opts)
-       
+        result = vmdk_ops.executeRequest(vm2_uuid, self.vm2_name, self.vm2_config_path, 'list', None, opts)        
         # listVMDK return vol name with the format like vol@datastore
         self.assertEqual(3, len(result))
-        self.assertEqual("tenant2_vol1@"+self.datastore_name, result[0]['Name'])
-        self.assertEqual("tenant2_vol2@"+self.datastore_name, result[1]['Name'])
-        self.assertEqual("tenant2_vol3@"+self.datastore_name, result[2]['Name'])
-    
+         
+        volume_names = ["tenant2_vol1@"+self.datastore_name,"tenant2_vol2@"+self.datastore_name,"tenant2_vol3@"+self.datastore_name]
+        for name in volume_names:
```

Comment:
It would be better to move this piece of code into verification util instead of copying multiple places. 

Please address the comment into your next diff.